### ISSUE TITLE
Sync conflicts are ignored and overwritten without warning

### ISSUE THREAD

**Issue by Terrox** [dmfs/opentasks#863]
I edited a task on my phone, then edited the task in Nextcloud. Once the phone synced with nextcloud the edits made on phone were gone without warning. This seems prone to problems with missing data and unexpected results.

**Comment by dmfs** [comment]
I agree this is annoying. However, sync conflicts must be handled by the sync app. OpenTasks doesn't know anything about the task status on the server.
Early next year I'm going to work on a new sync algorithm for your own apps which reduces the chances of a sync conflict because it syncs on a much finer granularity. This way we can avoid data loss in many situations when we're currently overriding data.

**Comment by returntrip** [comment]
I think this issue happened to me today. I believe what happened is that Davx5 was set to manual sync when I set the synch to daily/when local task is update, the up to date tasks in my phone were rewritten by the outdated tasks found on the Nextcloud server.

I guess there is no way to restore the local content?
